staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -321,7 +321,10 @@ func checkName(obj runtime.Object, name, namespace string, namer ScopeNamer) err
 //   interfaces
 func setObjectSelfLink(ctx context.Context, obj runtime.Object, req *http.Request, namer ScopeNamer) error {
 	if utilfeature.DefaultFeatureGate.Enabled(features.RemoveSelfLink) {
-		return nil
+		// TODO(#81191): Not setting selflink break CSI volume tests with alpha features,
+		// as they need to update the client-go dependency (to fix GetReferences function).
```

Comment:
we should probably backport the GetReferences fix as well

Reply:
You mean cherrypick it to previous branches?

Reply:
yes

Reply:
I opened: https://github.com/kubernetes/kubernetes/pull/81355 and https://github.com/kubernetes/kubernetes/pull/81407